packages/connect-examples/new-example/app/components/common/LogDisplay.tsx

```
@@ -0,0 +1,432 @@
+      error: 0,
+      request: 0,
+      response: 0,
+    };
+
+    logs.forEach((log) => {
+      if (
+        log &&
+        log.type &&
+        Object.prototype.hasOwnProperty.call(stats, log.type)
+      ) {
+        stats[log.type as keyof typeof stats]++;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)